test/test-Worker.js

```diff
@@ -194,6 +195,11 @@ test('Worker emits "died" if worker process died unexpectedly', async () =>
 
 test('worker process ignores PIPE, HUP, ALRM, USR1 and USR2 signals', async () =>
 {
+	if (os.platform() === 'win32') 
+	{
+		return;
+	}
```

Comment:
Why this?

BTW: Let's not use `{` `}` when the `if` condition takes a single line:

```js
if (os.platform() === 'win32') 
  return;
```

Reply:
Windowns has no `SIGPIPE/SIGALRM/SIGUSR1/SIGUSR2` signals,  and haven't found alternatives for `sigfillset` and `sigaction`. in `/worker/src/main.cc#IgnoreSignals` already use `_WIN32` marco to skip these logic code. so  in test, also skip.